webapp-deployment/values.yaml

```diff
@@ -1,16 +1,16 @@
 global:
-  name: "vault-demo"
+  name: "knox-retriever-demo"
   namespace: "c2053d"
   short-env: "dev"
 
 server:
   image:
-    registry: "artifacts.developer.gov.bc.ca"
-    context: "/cc20-gen-docker-local"
+    registry: "ghcr.io/bcgov/"
+    context: "nr-broker-openshift-knox-retriever"
```

Comment:
Should use artifactory (not ghcr.io directly)

Reply:
Use the remote artifactory link